pagopa/io-app
# feat: [#173962963] Expanded header icons touch area

## List of changes proposed in this pull request
The `hitSlop` was already being increased for a totale tappable area of `48` (`40` the size of the buttons, + `8` tolerance) but not on the horizontal axis.

## How to test
Buttons should be more easily tappable on the side. You can also increase `minTouchableAreaSize`, and add borders to buttons to verify the functionality more easily:

This is with `minTouchableAreaSize = 120`

https://user-images.githubusercontent.com/26572302/103284498-aaf85900-49db-11eb-98dc-44d27b8d5eed.mov
